vmware-base/src/com/cloud/hypervisor/vmware/util/VmwareHelper.java

```diff
@@ -598,6 +601,19 @@ public static void setBasicVmConfig(VirtualMachineConfigSpec vmConfig, int cpuCo
         vmConfig.setGuestId(guestOsIdentifier);
     }
 
+    public static void addUSBController(VirtualMachineConfigSpec vmConfigSpec) {
```

Comment:
While it's alright, you can make this method `addUSBController ` return something rather than modify the passed in VirtualMachineConfigSpec, and add a small unit test.